integrated_channels/blackboard/models.py

```diff
@@ -25,20 +25,20 @@ class BlackboardEnterpriseCustomerConfiguration(EnterpriseCustomerPluginConfigur
     client_id = models.CharField(
         max_length=255,
         null=True,
-        verbose_name="API Client ID",
+        verbose_name="API Client ID or Blackboard Application Key",
         help_text=(
             "The API Client ID provided to edX by the enterprise customer to be used to make API "
-            "calls on behalf of the customer."
+            "calls on behalf of the customer. Called Application Key in Blackboard"
         )
     )
 
     client_secret = models.CharField(
         max_length=255,
         null=True,
-        verbose_name="API Client Secret",
+        verbose_name="API Client Secret or Application Secret",
```

Comment:
similar situation as to above- seems like this would be either the client secret or the application secret.

Reply:
yup I guess it could confuse a bit, would it be okay if I cleaned this up in the next coming pr? only because changing this would need to regen migration (instead of creating a new one I would need to clean up and regen I think?)